vagrant/roles/logs.collect/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Create log directory structure
+  file:
+    path: "{{ config.statedir }}/{{ inventory_hostname }}"
+    state: directory
+  delegate_to: localhost
+
+- name: Copy /var/log directory
+  synchronize:
+    src: /var/log
+    dest: "{{ config.statedir }}/{{ inventory_hostname }}"
+    mode: pull
+
+- name: Process backend specific tasks
+  include_role:
+    name: 'sit.{{ config.be.name }}'
+    tasks_from: logs/main.yml
+
+- name: Copy journal
+  block:
+    - name: Extract journal since last boot
+      shell: journalctl -xb -n all > journal
+
+    - name: Copy journal file
+      fetch:
+        src: journal
+        dest: "{{ config.statedir }}/{{ inventory_hostname }}/log/"
+        flat: true
+
+- name: Copy dmesg
+  block:
+    - name: Extract dmesg contents
+      shell: dmesg -T > dmesg
```

Comment:
As in the previous comment, we can save it to /var/log/dmesg.kernel (for example) and let the other task to copy it.

Reply:
Done.